spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -230,6 +230,8 @@ class CodeEditor(TextEditBaseWidget):
     sig_bookmarks_changed = Signal()
     get_completions = Signal(bool)
     go_to_definition = Signal(str, int, int)
+    sig_go_to_uri = Signal(str)  # Used for testing
+    sig_uri_found = Signal(str)  # Used for testing

```

Comment:
Please add these signals below `sig_process_code_analysis` and explain more clearly when they are emitted and how they are used.